pkg/operator/ceph/object/user.go

```diff
@@ -226,3 +231,38 @@ func UnlinkUser(c *Context, id, bucket string) (string, int, error) {
 	}
 	return result, RGWErrorNone, nil
 }
+
+func EnableUserQuota(c *Context, id string) (string, int, error) {
```

Comment:
Is there a need to return the error code? Typically all the caller needs to check is `err != nil`. So I would expect to simplify these new functions to return `(string, error)`.

Reply:
@travisn : I tried to use the same format of [setUserQuota](https://github.com/rook/rook/blob/master/pkg/operator/ceph/object/user.go#L195) and [SetQuotaUserBucketMax](https://github.com/rook/rook/blob/master/pkg/operator/ceph/object/user.go#L185). Do you like make similar changes for those api's as well

Reply:
Done